mime/multipart.ts

```diff
@@ -259,6 +259,13 @@ export interface MultipartFormData {
   removeAll(): Promise<void>;
 }
 
+export interface ReadFormOptions {
+  maxMemory?: number;
+  dir?: string;
+  prefix?: string;
+  suffix?: string;
+}
```

Comment:
It might be great if these properties are documented using this syntax https://github.com/denoland/deno_std/blob/main/encoding/ascii85.ts#L5-L9

Reply:
[This commit](https://github.com/denoland/deno_std/pull/895/commits/5614f5f986d62000d99694de60a4f4486eace8d8) should fix this, but I don't know if this documentation is still subpar.